### ISSUE TITLE
Train/test & metrics interpretation

### ISSUE THREAD

**Issue by jyrr** [awslabs/gluon-ts#852]
Hi,

I'm new to time series prediction, so please excuse my limited knowledge and quite basal questions.

In GluonTS, following the tutorial, we create a train and test dataset, where the test dataset is the full dataset and the train dataset is the full dataset minus `prediction_length` values off the tail.

I'd like to compare my results to non-GluonTS solutions, for example LGBM (quantile regression), but I'm not sure how to. I've always learned to use a 80/20 split or something of the likes to split my train and test data due to overfitting etc. Are they simply not comparable?

Secondly, are the metrics calculated for the full test set or the last `prediction_length` values? Looking at the figure in the quick start tutorial, I notice that the median prediction is within the 90% confidence interval at each prediction, but the `Coverage[0.9]` value is ~88.7%. How do I interpret these?

**Comment by parimuns** [comment]
Hello 
I can just answer your one query 
" I notice that the median prediction is within the 90% confidence interval at each prediction, but the Coverage[0.9] value is ~88.7%"
Median prediction is always at p(50)(50% prediction interval ) and p(90) means 90% confidence interval.Moreover,if your model is learning good from data ,then its p(90) will be equal or greater than 90%,for less than 90% ,you need to tune the hyper-parameters.
